modules/org.restlet.test/src/org/restlet/test/ext/apispark/conversion/swagger/v2_0/Swagger2TestCase.java

```diff
@@ -212,20 +213,20 @@ protected Definition createDefinition() {
         return definition;
     }
 
-    private void compareMapOfStrings(Map<String, String> savedMap,
-            Map<String, String> translatedMap) {
-        if (savedMap != null || translatedMap != null) {
-            assertNotNull(savedMap);
-            assertNotNull(translatedMap);
-            for (Entry<String, String> entry : savedMap.entrySet()) {
-                assertEquals(savedMap.get(entry.getKey()),
-                        translatedMap.get(entry.getKey()));
-            }
-            for (Entry<String, String> entry : translatedMap.entrySet()) {
-                assertEquals(savedMap.get(entry.getKey()),
-                        translatedMap.get(entry.getKey()));
-            }
+    private void compareMapOfStrings(Map<String, Object> savedMap,
+            Map<String, Object> translatedMap) {
+        if (savedMap == null && translatedMap == null) {
```

Comment:
I do not disagree. ^^

It did my best at the local level, and did not explore the behavior of callers: do they really pass null in there? What for? How is it different from an empty map?

As a newcomer I'm still kind of slow to do stuff, so the schedule does allow me to go deeper this time, but I'll be glad to do so in the future. Unless you tell me to do it now, obviously.